ui/common/StatusBar.tsx

```diff
@@ -14,25 +15,34 @@ export default function StatusBar({
   next,
   input,
   expected,
+  beginMsg,
   successMsg,
   inProgressMsg,
   errorMsg,
   full,
+  hints,
 }: {
   next: string
   input: string
   expected: string
+  beginMsg?: string
```

Comment:
Can we change these to `beginMessage`, `successMessage`, etc? Let try to stay away from abbreviations at least for prop names.